jetty-fcgi/fcgi-client/src/main/java/org/eclipse/jetty/fcgi/client/http/HttpSenderOverFCGI.java

```diff
@@ -88,7 +88,7 @@ protected void sendHeaders(HttpExchange exchange, HttpContent content, Callback
         for (HttpField field : headers)
         {
             String name = field.getName();
-            String fcgiName = "HTTP_" + name.replaceAll("-", "_").toUpperCase(Locale.ENGLISH);
+            String fcgiName = "HTTP_" + name.replace('-', '_').toUpperCase(Locale.ENGLISH);
```

Comment:
We should probably have a StringUtil.replace for characters, even if it just calls the JVM, so we don't need to remember that char is OK to use direct but we should use StringUtil for string replacement.

Reply:
Fair enough.  I'll add it.

Reply:
Done, needs review